CHANGES

```diff
@@ -6,8 +6,8 @@
 
   *) The method for finding the storage location for the Windows RAND seed file
      has changed. First we check %RANDFILE%. If that is not set then we check
-     the directories %TMP%, %TEMP%, %USERPROFILE%, %SYSTEMROOT% and %HOME% in
-     that order. If all else fails we fall back to "C:".
+     the directories %HOME%, %USERPROFILE% and %SYSTEMROOT% in that order. If
+     all else fails we fall back to "C:".
```

Comment:
There is certain ambiguity in "fall back to C:". Thing is that C:file.ext refers to "current directory" on C: which is under program control. I mean if program changes it, then effective location for the file in question will do too. "Current directory" is maintained on per-letter basis. Maybe one can add few words to RAND_load_file.pod if this is actually what we want...